app/controllers/api/v1/sessions_controller.rb

```diff
@@ -0,0 +1,15 @@
+class Api::V1::SessionsController < ApplicationController
+  def create
+    user = User.find_by(email: user_params[:email])
+    if user.authenticate(user_params[:password])
+      render json: UsersSerializer.new(user)
```

Comment:
Wasn't sure if I should also be updating the session, but I think that would occur on the front-end.